.github/workflows/build_and_test.yml

```diff
@@ -20,7 +20,7 @@ jobs:
             pack_tool: tar -zcvf
             extension: .tar.gz
             pack_suffix: -C dist runner
-          - os: macos-10.15
+          - os: macos-11
```

Comment:
GHA is phasing out support for macos-10.15, so I pushed to 11.  It worked with zero issues.  I will probably add a macos-12 instance as well and see what happens.